BilibiliLive/Component/Video/VideoPlayerViewController.swift

```diff
@@ -422,9 +422,12 @@ extension VideoPlayerViewController {
             }
         }
         if let defaultRate = self.player?.defaultRate,
-           let speed = PlaySpeed.blDefaults.first(where: { $0.value == defaultRate }) {
+           let speed = PlaySpeed.blDefaults.first(where: { $0.value == defaultRate })
+        {
```

Comment:
This change is brought by swift-format. For last commit on this line, I forgot to run it before adding a commit.